Clinical-Genomics/scout
# Fix/severity text #2808

This PR fixes a bug: https://github.com/Clinical-Genomics/scout/issues/2808

This PR removes Span-tag for list items in Severity box, on a variant page, allowing for smoother rendering when base window is narrow.

**How to test**:
1. how to test it, possibly with real cases/data

Go to a cancer variant and make browser window narrow. Some lists of strings in the *Severity box* will render strangely. See picture.

![image](https://user-images.githubusercontent.com/1150065/131833786-3dc96a89-2b56-4a5c-8408-a8d8bfcec3e2.png)

2. Install patch, 

**Expected outcome**:
The functionality should be working. Try to narrow the browser page and the text should render smoothly and align within the box. However, way too narrow browser windows will always look strange.

![image](https://user-images.githubusercontent.com/1150065/131833482-6ff90dce-6805-4ca2-ab57-88b5ce9c10bd.png)

**Review:**
